src/app/utils/field-validation/validators/emailValidator.ts

```diff
@@ -21,7 +21,7 @@ type EmailValidatorConstructor = (
  */
 const emailFormatValidator: EmailValidator = (response) => {
   const { answer } = response
-  return isEmail(answer)
+  return isEmail(answer.trim())
```

Comment:
Hmm, might need an additional casting to string to prevent possible run-time errors? 🤔 
```suggestion
  return isEmail(String(answer).trim())
```

Reply:
this is already checked before this function is called, in`isProcessedSingleAnswerResponse`